ReactAndroid/src/main/java/com/facebook/react/uimanager/LayoutShadowNode.java

```diff
@@ -74,6 +74,7 @@ void setFromDynamic(Dynamic dynamic) {
   }
 
   private final MutableYogaValue mTempYogaValue;
+  private final Dynamic[] mPosition = new Dynamic[Spacing.INLINE_START + 1];
```

Comment:
Dynamic is probably heavier than we need here. Could keep these as `MutableYogaValue`?